app/lib/Plugins/SearchEngine/Elastic8/Query.php

```diff
@@ -257,8 +257,8 @@ public function rewriteSubquery($subquery) {
 				// 		ca_objects.dimensions_width:"30 cm",
 				// which is parsed as two terms ... "30", and "cm" to one relatively simple term query
 				if ($multiterm_all_terms_same_field && ($first_term = array_shift($terms))) {
-					$first_term = caRewriteElasticSearchTermFieldSpec($first_term);
 					$fld = $this->getFieldTypeForTerm($first_term);
+					$first_term = caRewriteElasticSearchTermFieldSpec($first_term);
```

Comment:
somehow doing the rewrite first only breaks this instance, even though we seem to do the same thing elsewhere. 

We might want to update getFieldTypeForTerm to work after the rewrite (eg. `ca_objects.idno` to `ca_objects\/idno`

Reply:
This does work in some cases, further investigation required.